calico_node/startup/startup.go

```diff
@@ -341,6 +341,22 @@ func validateIP(ipn *net.IPNet) {
 	warning("Unable to confirm IPv%d address %s is assigned to this host", ip.Version(), ip)
 }
 
+// evaluateENVBool evaluates a passed environment variable
+// Returns an boolean indicating if the parameter was defined
+// and an boolean if the argument was true or false
+// If it was false an additional log entry will be written
+
+func evaluateENVBool(envvar string, logentry string) (isDefined bool, isValue bool ) {
```

Comment:
I think there could be a better name for the 'isValue' variable.  Maybe envValue or possibly just value, or the extra verbose option envBoolValue.